### PR TITLE
Remove fs dependency in favor of file_system

### LINKED ISSUE #68: Docker support?
I'm running my Phoenix app inside Docker and wondering if there's any way to get `test.watch` to work inside Docker?

It runs fine once, but doesn't detect filesystem changes.  I have the host application folder shared with the Container, so it should pick up the changes (live reload works with the Phoenix web server).

Has anyone tried this?

### PR DESCRIPTION
**Pull request by Papipo**
Fixes #68 with `file_system` instead of `fs`.

### CODE REVIEW

**Comment by Papipo** [comment]
BTW, I was using `brew` installed docker commands and I had to switch to Docker for Mac. That plus these changes make `mix test.watch` flawlessly.

File: lib/mix_test_watch/watcher.ex [hunk 1]
```diff
@@ -27,8 +29,17 @@ defmodule MixTestWatch.Watcher do
...
-    :ok = :fs.subscribe()
-    {:ok, []}
+    opts = [dirs: [Path.absname("")], name: :mix_test_watcher]
+    case FileSystem.start_link(opts) do
+      {:ok, _} ->
+        FileSystem.subscribe(:mix_test_watcher)
+        {:ok, []}
+      other ->
+        Logger.warn """
+        Could not start the file system monitor.
```
Comment (lpil) [inline lib/mix_test_watch/watcher.ex]: Do we want to warn here or should we just crash? Is it possible to continue at all if the file system watcher doesn't work?

Comment (Papipo) [reply lib/mix_test_watch/watcher.ex]: Shouldn't this crash? it's returning something else than `{:ok, _}`, right?

**Comment by mhodgson** [comment]
Thanks for this! Confirmed this fix works out of the box with Docker for Mac. Any plans to cut a release?

**Comment by lpil** [comment]
There you go!

**Pull request merged**